docs: 7.4 what's new (Add expressions note) (#30446)
grafana/grafana: docs/sources/whatsnew/whats-new-in-v7-4.md

The main use case is for [multi-dimensional](https://grafana.com/docs/grafana/latest/getting-started/timeseries-dimensions/#time-series-dimensions) data sources used with the upcoming next generation alerting, but expressions can be used with backend data sources and visualization as well.

> > **Note:** Queries built with this feature may break with minor version upgrades until Grafana 8 is released. This feature also does not work with current Grafana alerting.

Reviewer: ```suggestion
> **Note:** Queries built with this feature may break with minor version upgrades until Grafana 8 is released. This feature also does not work with the current Grafana alerting.
```

> > **Note:** Queries built with this feature may break with minor version upgrades until Grafana 8 is released. This feature also does not work with the current Grafana alerting.

Reviewer 2: ```suggestion
> **Note:** Queries built with this feature might break with minor version upgrades until Grafana 8 is released. This feature does not work with the current Grafana alerting.
```
